doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -865,6 +865,30 @@ Controlling display
    interpreted from left to right, so in case of an overlap, the flags on the
    right have higher priority, meaning that `A,-A` is equivalent to `-A`.
 
+.. opt:: Debug "{+, @ident {? = num } }"
+
+   This option configures the display of debug messages. Each debug
+   message has a named component and a verbosity. When the debug level
+   of the component is set at or higher than the verbosity of the
+   message, the message is printed.
+
+   In most cases, you do not need to know the verbosity of a message:
+   ``Set Debug "component".`` will set it to the level of the main
+   messages in the component.
+
+   A list of components and their current levels can be obtained
+   through ``Test Debug.``.
+
+   The component ``all`` is special and sets all other components to
+   its level. Its default level is ``1``, where most components
+   default to ``2``, so if you really want messages from all
```

Comment:
Why not make its default `1` as well then?

Reply:
While we still have the legacy messages I think we don't want -debug to make everything start spamming